packages/reference/src/common/useEditorPermissions.ts

```diff
@@ -50,29 +50,25 @@ export function useEditorPermissions(props: EditorPermissionsProps) {
     }
 
     async function checkLinkAccess() {
-      if (props.allContentTypes?.length) {
-        if (entityType === 'Asset') {
-          const canRead = await canOnEntity('read', 'Asset');
-          setCanLinkEntity(canRead);
-        }
-        if (entityType === 'Entry') {
-          setCanLinkEntity(readableContentTypes.length > 0);
-        }
-      } else {
-        setCanLinkEntity(true)
+      if (entityType === 'Asset') {
+        const canRead = await canOnEntity('read', 'Asset');
+        setCanLinkEntity(canRead);
+      }
+      if (entityType === 'Entry') {
+        setCanLinkEntity(readableContentTypes.length > 0);
       }
     }
 
     void checkLinkAccess();
-  }, [entityType, parameters.instance, readableContentTypes, props.allContentTypes]);
+  }, [entityType, parameters.instance, readableContentTypes]);
```

Comment:
Looks like we don't need props.allContentTypes in this component anymore, right?

Reply:
You are right! good point, then I'll delete it from the **EditorPermissionsProps** interface and I'll make sure that we are not passing it in the  props anymore 👍

Reply:
I tried to delete props.allContentType from the EditorPermissionsProps, and I found that we are passing them to **useContentTypePermissions** hook